templates/templated_email/answer_notification.html

```diff
@@ -0,0 +1,43 @@
+<html lang="pl-PL">
+<head>
+   <meta charset="UTF-8">
+
+    <body style=" size: A5;">
+        <div style="background-color: #0d5bdd; text-align: center; padding-top: 20px;
+        padding-bottom: 20px; border-top-left-radius: 10px; border-top-right-radius: 10px">
+            <h1 style="color: #343a40">
+                Wysłano odpowiedź na formularz
+            </h1>
+        </div>
+
+
+        <div style="background-color: #1aa3e8; padding-top: 10px; padding-bottom: 10px">
+            <div style="margin-left: 20px">
+                <h2>Witaj {% if user.first_name and user.first_name|length %}{{ user.first_name }}{% else %}{{ user.username }}{% endif %},</h2>
```

Comment:
This field `user.first_name` and/or `user.username` used to be empty
and the reason for this is in "onboarding/mailing.py"